packages/teleterm/package.json

```diff
@@ -17,7 +17,8 @@
     "build-main": "webpack build --config webpack.main.config.js --progress --mode=production",
     "build-renderer": "webpack build --config webpack.renderer.prod.config.js --progress",
     "build-natives": "electron-builder install-app-deps",
-    "package": "electron-builder build --publish never -c.extraMetadata.name=teleconnect"
+    "package": "electron-builder build --publish never -c.extraMetadata.name=teleconnect",
+    "generate-grpc-shared": "npx -y --target_arch=x64 --package=grpc_tools_node_protoc_ts --package=grpc-tools -- grpc_tools_node_protoc -I=src/sharedProcess/api/proto --ts_out=service=grpc-node,mode=grpc-js:src/sharedProcess/api/protogen --grpc_out=grpc_js:src/sharedProcess/api/protogen --js_out=import_style=commonjs,binary:src/sharedProcess/api/protogen src/sharedProcess/api/proto/*.proto"
```

Comment:
> The script command is really long, but I didn't find any way to extract a npx script to something like a file :/ I hope we will be able to get rid of it when `grpc-tools` will have a version for M1.

You could put it in a bash file, but I think what we got is good enough for now.